src/components/homepage/footer/footer.tsx

```diff
@@ -185,16 +185,12 @@ const Footer = (): JSX.Element => {
 
             <div className="flex flex-row gap-5">
               <a
-                href="https://github.com/healthyregions"
-                title="@heathyregions on GitHub"
+                href="https://github.com/healthyregions/SDOHPlace"
+                title="@sdohplace on GitHub"
```

Comment:
See other comment.

Reply:
Fixed in [4f300b0](https://github.com/healthyregions/SDOHPlace/pull/50/commits/4f300b0bf06615fa1e1144a4b49d8929cb11b591)